linera-core/src/local_node.rs

```diff
@@ -71,6 +73,12 @@ pub enum LocalNodeError {
 
     #[error("Blobs not found: {0:?}")]
     BlobsNotFound(Vec<BlobId>),
+
+    #[error("A validator requested blobs that are not required: {0:?}")]
+    UnrequiredBlobsRequested(Vec<BlobId>),
```

Comment:
Then that's not a `LocalNodeError` but a `NodeError`. I tried to move these things to the right place in https://github.com/linera-io/linera-protocol/pull/2895; let's not mix them up again.